docs/fsm.rst

```diff
@@ -131,6 +131,14 @@ secondary will be online to accept writes. When the old primary reaches
 the wait_primary state then the secondary is safe to take offline with
 minimal consequences.
 
+**Prepare_maintenance**
```

Comment:
`wait_maintenance` state should alsoget a description here. While reading through the list I also realised that `prepare_promotion` is not in there either. Probably good to add it.

Reply:
Added, thanks!